lib/unionlabs-primitives/src/fixed_bytes.rs

```diff
@@ -236,6 +236,32 @@ impl<'de, const BYTES: usize, E: Encoding> serde::Deserialize<'de> for FixedByte
     }
 }
 
+#[cfg(feature = "schemars")]
+impl<const BYTES: usize, E: Encoding> schemars::JsonSchema for FixedBytes<BYTES, E> {
+    fn schema_name() -> String {
+        "Hash".to_string()
+    }
+
+    /// The `FixedBytes` object is serialized as an array in JSON
```

Comment:
it's serialized as string according to `E` (hex, hex unprefixed, base64, etc). jsonschema for FixedBytes and Bytes are a bit more complicated if you want full correctness, but for now just make them be strings